src/lwc/rdActiveSchedule/rdActiveSchedule.css

```diff
@@ -0,0 +1,4 @@
+/* This is intended to hide the menu dropdown-arrows in the column headers. It doesn't work */
```

Comment:
If this still isn't working, and not being brought up as an issue during testing, I don't think this file is needed in this new component.